src/__tests__/missions.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import Missions from '../components/Missions';
+import missionsReducer from '../redux/missions/missionsSlice';
+
+const missionStore = configureStore({
+  reducer: {
+    missions: missionsReducer,
+  },
+});
+
+const data = [
+  {
+    mission_id: 'mission1',
+    mission_name: 'Mission Apolo',
+    description: 'Mission Mission, Mission Mission Space',
+  },
+  {
+    mission_id: 'mission2',
+    mission_name: 'Mission Apolo 2',
+    description: 'Mission Mission, Mission Mission Space 2',
+  },
+  {
+    mission_id: 'mission3',
+    mission_name: 'Mission Apolo 3',
```

Comment:
- [ ] Great job working on the tests but I think you have failed tests I could show you the error message and please try to check on the issue and solve it ⬇🔥⭐It's good practice to have passed tests for all the methods to ensure that they work as expected.. Kindly make sure to test it 💪

1-
![image](https://github.com/anyars-encarta/react-group-project/assets/58553711/c5a4d7af-b637-4fea-b06d-011609eb1808)

2-
![image](https://github.com/anyars-encarta/react-group-project/assets/58553711/7196083e-d53e-4cc2-a467-5d8725c4d70c)

3- 
![image](https://github.com/anyars-encarta/react-group-project/assets/58553711/9833c308-f82e-424f-aa16-9750c06a2ddc)

4-

![image](https://github.com/anyars-encarta/react-group-project/assets/58553711/fd7b886f-dd5c-49e3-952c-1adc7159ea19)

Reply:
This was very unexpected. We shall look into it asap